parser/package.json

```diff
@@ -20,7 +20,7 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
-    "expensify-common": "Expensify/expensify-common#a018512b1e0fce4d4b5a61c00ca826b8887007ad",
+    "expensify-common": "2.0.4",
```

Comment:
Can we update to the latest (current latest is 2.0.9)?

Reply:
@tomekzaw I just updated to the latest version. Please help to check the patch again, I see that we have a logger on the latest version.

Reply:
Thanks for bumping, will review it now.